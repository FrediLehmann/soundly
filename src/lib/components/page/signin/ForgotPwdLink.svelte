<script type="ts" context="module">
  import { Link } from '$lib/components/atoms';
</script>

<Link href="/forgotpassword" class="mbs-2">Forgot password?</Link>
