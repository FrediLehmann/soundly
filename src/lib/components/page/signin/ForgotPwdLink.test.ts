/**
 * @jest-environment jsdom
 */

import '@testing-library/jest-dom';
import { render } from '@testing-library/svelte';

import ForgotPwdLink from './ForgotPwdLink.svelte';

describe('<ForgotPwdLink /> component', () => {
  test('renders correctly', () => {
    const { component } = render(ForgotPwdLink);
    expect(component).toMatchSnapshot();
  });
});
