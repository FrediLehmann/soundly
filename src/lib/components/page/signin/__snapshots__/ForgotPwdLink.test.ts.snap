// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ForgotPwdLink /> component renders correctly 1`] = `
ForgotPwdLink {
  "$$": Object {
    "after_update": Array [],
    "before_update": Array [],
    "bound": Object {},
    "callbacks": Object {},
    "context": Map {},
    "ctx": Array [],
    "dirty": Array [
      -1,
    ],
    "fragment": Object {
      "c": [Function],
      "d": [Function],
      "i": [Function],
      "l": [Function],
      "m": [Function],
      "o": [Function],
      "p": [Function],
    },
    "not_equal": [Function],
    "on_destroy": Array [
      [Function],
    ],
    "on_disconnect": Array [],
    "on_mount": Array [],
    "props": Object {},
    "root": <div>
      <a
        class="mbs-2 svelte-1u7l9fe"
        href="/forgotpassword"
        sveltekit:prefetch="true"
      >
        Forgot password?
      </a>
    </div>,
    "skip_bound": false,
    "update": [Function],
  },
  "$capture_state": [Function],
}
`;
